ucsb-cs156-w24/team03-w24-5pm-4
# Create Page for RecommendationRequest plus tests and stories

Create Page for RecommendationRequest done. All test and coverages pass.

Storybook: https://ucsb-cs156-w24.github.io/team03-w24-5pm-4/prs/81/storybook/?path=/story/pages-recommendationrequest-recommendationrequestcreatepage--default

Steps for testing:
1. Log In
2. Click on Recommendation Requests on Navbar 
<img width="1319" alt="Screenshot 2024-02-13 at 11 22 00 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-4/assets/91992519/0f656c21-e57a-4f6b-a2a8-7bd1aa7fec49">
3. Click Create
<img width="1317" alt="Screenshot 2024-02-13 at 11 22 45 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-4/assets/91992519/f1c76041-a370-42cf-a6d8-a9d9b1026988">
4. Enter in fields correctly
<img width="1310" alt="Screenshot 2024-02-13 at 11 23 45 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-4/assets/91992519/2ba55cd8-911c-473e-b9b7-9451dd2845ed">
5. Click Create
<img width="1599" alt="Screenshot 2024-02-13 at 11 24 16 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-4/assets/91992519/d1e9b10d-286c-41d4-a165-a6388c2cb2a5">
6. Check on Swagger if entry created
<img width="1423" alt="Screenshot 2024-02-13 at 11 26 11 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-4/assets/91992519/b326f647-905f-4032-9726-3e6fe6d2dec2">

Closes #31
